crm_claim_rma/tests/test_lp_1282584.py

```diff
@@ -78,7 +73,7 @@ def test_00(self):
 
         res = self.WizardMakePicking.action_create_picking(
             cr, uid, [wizard_id], context=wiz_context)
-        self.assertEquals(res.get('res_model'), 'stock.picking.in',
+        self.assertEquals(res.get('res_model'), 'stock.picking',
                           "Wrong model defined")
```

Comment:
This check seems meaningless to me. Here we tested the wizard was opening the right view based on model.
We cannot rely on model anymore.